libafl/src/stages/calibrate.rs

```diff
@@ -71,6 +71,8 @@ pub struct CalibrationStage<C, O, OT, S> {
     stage_max: usize,
     /// If we should track stability
     track_stability: bool,
+    /// If we should send the default stability
+    need_default_stability: bool,
```

Comment:
This needs to be part of the metadata, else the stage will always resend 100% after a crash.
Probably the best way would be to always create the `UnstableEntriesMetadata`, and then only send 100% if it didn't exist before - so use its existence as flag

Reply:
(explanation: on crash the InProcessExecutor will restart/refork the program and reload the state - anything that's not in the state (metadtata lives in the state) will be reset to the initial value, accordingly